tests/jest.back-end.integration.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from "@jest/types";
+import unit from "./jest.back-end.unit";
+
+const config: Config.InitialOptions = {
+  ...unit,
+  roots: ["./tests/back-end/integration"],
+  displayName: "back-end:integration",
+  setupFilesAfterEnv: ["<rootDir>/tests/back-end/setup.jest.ts"],
+  globalSetup: "<rootDir>/tests/utils/global-setup.ts",
+  globalTeardown: "<rootDir>/tests/utils/global-teardown.ts"
+};
```

Comment:
It looks like this handles the setup and teardown of the dockerized app for the integration tests. Would it make sense to remove the `scripts/run-integration-tests.sh` script and adjust the `package.json` to directly call jest with this config now?

Reply:
Ah yeah, missed that. I'll remove it in my next PR